server/src/test/java/org/apache/cassandra/sidecar/routes/cassandra/JmxCommonTest.java

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
This file is same as in other PR https://github.com/apache/cassandra-sidecar/pull/159/files. Either one of these PRs will merge this file